### ISSUE TITLE
[Bug] GP Values Inconsistency When Shared

### ISSUE THREAD

**Issue by Jh16** [Alumian/CEPGP-Retail#153]
**Describe the bug**
*A clear and concise description of what the bug is.*
When entering the override values for item GP. Then trying to share that data. Even tried sending the settings file at the following location:
`World of Warcraft\_classic_\WTF\Account\<AccountName>\SavedVariables\CEPGP.lua`
The problem is it will give only one or two items those GP values then continue with the formula for everything else.

To clarify, the character that enters the override data works fine.
Any character the data is shared to, it doesn't use the override data.

Also noticed when someone tried to import data from me, my GP values also screwed up.

**To Reproduce**
*How can this bug be reproduced?*
Type in multiple override values for GP. Then switch to a different character and cursor over those same items in Atlasloot. It won't be the same value that it was assigned despite it being in the override settings.

![WoWScrnShot_120719_180311](https://user-images.githubusercontent.com/5688177/70382163-9baff480-191c-11ea-9f13-976c06753cde.jpg)
![WoWScrnShot_120719_180325](https://user-images.githubusercontent.com/5688177/70382164-9baff480-191c-11ea-8ba4-d5a67614c77a.jpg)
![WoWScrnShot_120719_180338](https://user-images.githubusercontent.com/5688177/70382165-9baff480-191c-11ea-980a-b4b817fa414c.jpg)

**Debug Information**
*Type /cepgp debug and copy and paste the information into this bug report.*
<details><summary>Debug Info</summary><br />
Version: 1.12.7<br />
Locale: enUS<br />
GP Formula: (1x(1^<sup>((ilvl/26)+(rarity-4))</sup>)xSlot Modifier)x1<br />Base GP: 10<br />
Base GP Decay Factor: false<br />
Standby EP: False<br />
Standby Offline: False<br />
Standby Percent: 0<br />
Standby Accept Whispers: True<br />
Standby EP by Rank: False<br />
Standby EP Manual Delegation: True<br />
GUI for Loot: False<br />
Loot Response Keyphrase: !bid<br />
Standby EP Whisper Keyphrase: !standby<br />
Auto Pass on Ineligible Items: true<br />
Full Raid Loot Visibility: true<br />
GP on Tooltips: true<br />
Suppress Loot Announcements: false<br />
Minimum EP: false, 0<br />
Reporting Channel: Guild<br />
Loot Response Channel: RAID<br />
Minimum Threshold for Loot: 4<br />
Allow Force Sync: false, <br />
<br /><details><summary>Auto EP</summary>
High Priestess Jeklik: False<br />
Gothik the Harvester: False<br />
Buru the Gorger: False<br />
Wushoolay: False<br />
Lord Kri: False<br />
Emperor Vek'nilash: False<br />
Ossirian the Unscarred: False<br />
Golemagg the Incinerator: False<br />
Grand Widow Faerlina: False<br />
High Priestess Mar'li: False<br />
Majordomo Executus: False<br />
High Priest Venoxis: False<br />
Gehennas: False<br />
Taerar: False<br />
Ysondre: False<br />
Thane Korth'azz: False<br />
High Priest Thekal: False<br />
Noth the Plaguebringer: False<br />
Lethon: False<br />
Moam: False<br />
The Prophet Skeram: False<br />
Flamegor: False<br />
C'Thun: False<br />
General Rajaxx: False<br />
Princess Huhuran: False<br />
Thaddius: False<br />
Loatheb: False<br />
Lady Blaumeux: False<br />
Highlord Mograine: False<br />
Heigan the Unclean: False<br />
Azuregos: False<br />
Doom Lord Kazzak: False<br />
Teremus the Devourer: False<br />
Jin'do the Hexxer: False<br />
Baron Geddon: False<br />
Renataki: False<br />
Sulfuron Harbinger: False<br />
Gri'lek: False<br />
Bloodlord Mandokir: False<br />
Patchwerk: False<br />
Vaelastrasz the Corrupt: False<br />
Gluth: False<br />
Sapphiron: False<br />
Emeriss: False<br />
Onyxia: False<br />
Princess Yauj: False<br />
Kurinnaxx: False<br />
Shazzrah: False<br />
Ebonroc: False<br />
Battleguard Sartura: False<br />
Hakkar: False<br />
Kel'Thuzad: False<br />
Nefarian: False<br />
Fankriss the Unyielding: False<br />
Lucifron: False<br />
Anub'Rekhan: False<br />
Hazza'rah: False<br />
Viscidus: False<br />
Broodlord Lashlayer: False<br />
Ayamiss the Hunter: False<br />
Ragnaros: False<br />
Maexxna: False<br />
Ouro: False<br />
Vem: False<br />
Grobbulus: False<br />
Instructor Razuvious: False<br />
Razorgore the Untamed: False<br />
Magmadar: False<br />
Sir Zeliek: False<br />
Gahz'ranka: False<br />
Emperor Vek'lor: False<br />
High Priestess Arlokk: False<br />
Firemaw: False<br />
Chromaggus: False<br />
Garr: False<br />
</details>
<details><summary>EP Values</summary>
High Priestess Jeklik: 0<br />
Gothik the Harvester: 5<br />
Buru the Gorger: 0<br />
Wushoolay: 0<br />
Lord Kri: 5<br />
Emperor Vek'nilash: 5<br />
Ossirian the Unscarred: 0<br />
Golemagg the Incinerator: 5<br />
Grand Widow Faerlina: 5<br />
High Priestess Mar'li: 0<br />
Majordomo Executus: 5<br />
High Priest Venoxis: 0<br />
Gehennas: 5<br />
Taerar: 0<br />
Ysondre: 0<br />
Thane Korth'azz: 5<br />
High Priest Thekal: 0<br />
Noth the Plaguebringer: 5<br />
Lethon: 0<br />
Moam: 0<br />
The Prophet Skeram: 5<br />
Flamegor: 5<br />
C'Thun: 5<br />
General Rajaxx: 0<br />
Princess Huhuran: 5<br />
Thaddius: 5<br />
Loatheb: 5<br />
Lady Blaumeux: 5<br />
Highlord Mograine: 5<br />
Heigan the Unclean: 5<br />
Azuregos: 0<br />
Doom Lord Kazzak: 0<br />
Teremus the Devourer: 0<br />
Jin'do the Hexxer: 0<br />
Baron Geddon: 5<br />
Renataki: 0<br />
Sulfuron Harbinger: 5<br />
Gri'lek: 0<br />
Bloodlord Mandokir: 0<br />
Patchwerk: 5<br />
Vaelastrasz the Corrupt: 5<br />
Gluth: 5<br />
Sapphiron: 5<br />
Emeriss: 0<br />
Onyxia: 0<br />
Princess Yauj: 5<br />
Kurinnaxx: 0<br />
Shazzrah: 5<br />
Ebonroc: 5<br />
Battleguard Sartura: 5<br />
Hakkar: 0<br />
Kel'Thuzad: 5<br />
Nefarian: 5<br />
Fankriss the Unyielding: 5<br />
Lucifron: 5<br />
Anub'Rekhan: 5<br />
Hazza'rah: 0<br />
Viscidus: 5<br />
Broodlord Lashlayer: 5<br />
Ayamiss the Hunter: 0<br />
Ragnaros: 5<br />
Maexxna: 5<br />
Ouro: 5<br />
Vem: 5<br />
Grobbulus: 5<br />
Instructor Razuvious: 5<br />
Razorgore the Untamed: 5<br />
Magmadar: 5<br />
Sir Zeliek: 5<br />
Gahz'ranka: 0<br />
Emperor Vek'lor: 5<br />
High Priestess Arlokk: 0<br />
Firemaw: 5<br />
Chromaggus: 5<br />
Garr: 5<br />
</details>
<details><summary>Standby Guild Ranks</summary>
0: False<br />
1: False<br />
2: False<br />
3: False<br />
4: False<br />
5: False<br />
6: False<br />
7: False<br />
8: False<br />
9: False<br />
</details>
<details><summary>Slot Weights</summary>
2HWEAPON: 0<br />
NECK: 0<br />
HEAD: 0<br />
WEAPON: 0<br />
WRIST: 0<br />
WEAPONMAINHAND: 0<br />
SHIELD: 0<br />
HOLDABLE: 0<br />
RANGED: 0<br />
THROWN: 0<br />
FEET: 0<br />
RELIC: 0<br />
RANGEDRIGHT: 0<br />
WEAPONOFFHAND: 0<br />
TRINKET: 0<br />
FINGER: 0<br />
CLOAK: 0<br />
CHEST: 0<br />
ROBE: 0<br />
HAND: 0<br />
WAIST: 0<br />
EXCEPTION: 0<br />
LEGS: 0<br />
SHOULDER: 0<br />
</details>
<details><summary>Override List</summary>
[Fire Runed Grimoire]: 12<br />
[Nightslayer Belt]: 2<br />
[Perdition's Blade]: 40<br />
[Salamander Scale Pants]: 30<br />
[Nightslayer Cover]: 12<br />
[Earthfury Legguards]: 12<br />
[Felheart Pants]: 12<br />
[Nightslayer Chestpiece]: 16<br />
[Cloak of the Shrouded Mists]: 20<br />
[Earthshaker]: 12<br />
[Cenarion Bracers]: 2<br />
[Bindings of the Windseeker]: 2<br />
[Flameguard Gauntlets]: 14<br />
[Lawbringer Boots]: 0<br />
[Sash of Whispered Secrets]: 16<br />
[Mantle of Prophecy]: 12<br />
[Bloodfang Pants]: 32<br />
[Bindings of the Windseeker]: 2<br />
[Earthfury Epaulets]: 12<br />
[Cauterizing Band]: 28<br />
[Giantstalker's Belt]: 2<br />
[Shadowstrike]: 0<br />
[Netherwind Pants]: 32<br />
[Robes of Prophecy]: 16<br />
[Onslaught Girdle]: 48<br />
[Manastorm Leggings]: 4<br />
[The Eye of Divinity]: 30<br />
[Blastershot Launcher]: 14<br />
[Azuresong Mageblade]: 28<br />
[Vambraces of Prophecy]: 2<br />
[Arcanist Crown]: 12<br />
[Legplates of Ten Storms]: 32<br />
[Lawbringer Helm]: 0<br />
[Lawbringer Spaulders]: 0<br />
[Magma Tempered Boots]: 0<br />
[Circlet of Prophecy]: 12<br />
[Gutgore Ripper]: 12<br />
[Legplates of Wrath]: 32<br />
[Giantstalker's Boots]: 12<br />
[Cenarion Boots]: 12<br />
[Arcanist Leggings]: 12<br />
[Arcanist Robes]: 16<br />
[Leggings of Transcendence]: 32<br />
[Felheart Robes]: 16<br />
[Arcanist Bindings]: 2<br />
[Mana Igniting Cord]: 28<br />
[Nemesis Leggings]: 32<br />
[Flamewaker Legplates]: 4<br />
[Nightslayer Gloves]: 12<br />
[Cenarion Belt]: 2<br />
[Cenarion Spaulders]: 12<br />
[Earthfury Gauntlets]: 12<br />
[Deep Earth Spaulders]: 10<br />
[Cenarion Vestments]: 16<br />
[Core Forged Greaves]: 4<br />
[Earthfury Bracers]: 2<br />
[Ring of Spell Power]: 12<br />
[Medallion of Steadfast Might]: 36<br />
[Obsidian Edged Blade]: 20<br />
[Seal of the Archmagus]: 4<br />
[Choker of Enlightenment]: 8<br />
[Aurastone Hammer]: 18<br />
[Cenarion Leggings]: 12<br />
[Eskhandar's Right Claw]: 16<br />
[Dragon's Blood Cape]: 16<br />
[Wristguards of Stability]: 12<br />
[Earthfury Belt]: 2<br />
[Felheart Bracers]: 2<br />
[Pants of Prophecy]: 12<br />
[Helm of Might]: 12<br />
[Boots of Prophecy]: 12<br />
[Sorcerous Dagger]: 16<br />
[Breastplate of Might]: 16<br />
[Arcanist Gloves]: 12<br />
[Quick Strike Ring]: 36<br />
[Sabatons of the Flamewalker]: 12<br />
[Bonereaver's Edge]: 38<br />
[Nightslayer Boots]: 12<br />
[Giantstalker's Helmet]: 12<br />
[Heavy Dark Iron Ring]: 30<br />
[Stormrage Legguards]: 32<br />
[Lawbringer Gauntlets]: 0<br />
[Band of Sulfuras]: 16<br />
[Earthfury Vestments]: 16<br />
[Girdle of Prophecy]: 2<br />
[Giantstalker's Gloves]: 12<br />
[Earthfury Boots]: 12<br />
[Bracers of Might]: 2<br />
[Lawbringer Legplates]: 0<br />
[Pauldrons of Might]: 12<br />
[Aged Core Leather Gloves]: 16<br />
[Belt of Might]: 2<br />
[Arcanist Boots]: 12<br />
[Arcanist Mantle]: 12<br />
[Nightslayer Pants]: 12<br />
[Eye of Sulfuras]: 4<br />
[Cenarion Gloves]: 12<br />
[Wristguards of True Flight]: 28<br />
[Lawbringer Bracers]: 0<br />
[Nightslayer Bracelets]: 2<br />
[Essence of the Pure Flame]: 8<br />
[Giantstalker's Bracers]: 2<br />
[Arcanist Belt]: 2<br />
[Talisman of Ephemeral Power]: 36<br />
[Earthfury Helmet]: 12<br />
[Lawbringer Chestguard]: 0<br />
[Ancient Petrified Leaf]: 36<br />
[Gloves of Prophecy]: 12<br />
[Wild Growth Spaulders]: 36<br />
[Fireproof Cloak]: 4<br />
[Helm of the Lifegiver]: 8<br />
[Striker's Mark]: 40<br />
[Crimson Shocker]: 1<br />
[Giantstalker's Breastplate]: 16<br />
[Drillborer Disk]: 24<br />
[Brutality Blade]: 24<br />
[Dragonstalker's Legguards]: 32<br />
[Nightslayer Shoulder Pads]: 12<br />
[Giantstalker's Epaulets]: 12<br />
[Legplates of Might]: 12<br />
[Felheart Horns]: 12<br />
[Staff of Dominance]: 20<br />
[Robe of Volatile Power]: 28<br />
[Sabatons of Might]: 12<br />
[Felheart Gloves]: 12<br />
[Gauntlets of Might]: 12<br />
[Spinal Reaper]: 26<br />
[Band of Accuria]: 48<br />
[Gloves of the Hypnotic Flame]: 4<br />
[Felheart Shoulder Pads]: 12<br />
[Judgement Legplates]: 0<br />
[Crown of Destruction]: 40<br />
[Core Hound Tooth]: 30<br />
[Fireguard Shoulders]: 4<br />
[Finkle's Lava Dredger]: 8<br />
[Felheart Slippers]: 12<br />
[Shard of the Flame]: 4<br />
[Cenarion Helm]: 12<br />
[Giantstalker's Leggings]: 12<br />
[Lawbringer Belt]: 0<br />
[Choker of the Fire Lord]: 42<br />
[Tome of Tranquilizing Shot]: 0<br />
[Malistar's Defender]: 14<br />
[Felheart Belt]: 2<br />
</details>
<details><summary>Addons List</summary>
AdiBags<br />
AdvancedInterfaceOptions<br />
Atlas<br />
Atlas_ClassicWoW<br />
AtlasLootClassic<br />
Auctionator<br />
AuctionSold<br />
AutoTrackQuests<br />
BagFreeSpaceCounter<br />
BetterVendorPrice<br />
BlueShamans<br />
ClassTrainerPlus<br />
CEPGP<br />
Classic Quest Log<br />
ClassicAuraDurations<br />
ClassicCastbars<br />
ClassicCastbars_Options<br />
ClassicCodex<br />
Doom_CooldownPulse<br />
ExRT<br />
GTFO<br />
Guild_Roster_Manager<br />
HHTD<br />
idTip<br />
InFlight_Load<br />
ItemTooltipProfessionIcons<br />
Leatrix_Maps<br />
Leatrix_Plus<br />
LetMeCast<br />
MacroTalkClassic<br />
MaxCam<br />
Memoria<br />
MerInspect<br />
MonkeySpeed<br />
NeatPlates<br />
NeatPlatesHub<br />
NeatPlatesWidgets<br />
NeatPlates_Alvara<br />
NeatPlates_Graphite<br />
NeatPlates_Grey<br />
NeatPlates_Neon<br />
NeatPlates_Quatre<br />
NeatPlates_Roth<br />
NeatPlates_Slim_Horizontal<br />
NeatPlates_Slim_Vertical<br />
NeatPlates_BlizzardPlates<br />
NeatPlates_ClassicPlates<br />
NeatPlates_Simple<br />
NeatPlates_Renaitre<br />
Necrosis<br />
OmniCC<br />
oRA3<br />
orbSellAndRepair<br />
Prat-3.0<br />
Prat-3.0_Libraries<br />
out of range 1.3<br />
QuestIconDesaturation<br />
Quester<br />
QuestFrameFixer<br />
QuickTargets<br />
RealMobHealth<br />
RepHelper_Classic<br />
Sheepdog<br />
SimplyGInv<br />
SoulSort<br />
Spy<br />
TinyTooltip<br />
WeaponSwingTimer<br />
WhatsTraining<br />
WIM<br />
XandY<br />
XPTracker<br />
YouGotMail<br />
SpeedyAutoLoot<br />
DBM-Core<br />
DBM-DefaultSkin<br />
DBM-StatusBarTimers<br />
DBM-VPVEM<br />
</details>
<details><summary>Enabled Plugins</summary>
</details>
</details>

**Comment by Jh16** [comment]
Now the values are going on the fritz on the character that I entered the override values.

**Comment by Alumian** [comment]
Just to clarify, when you say on the fritz, do you mean it's exhibiting the same behaviour as you described in your initial report, or is the behaviour different?

**Comment by Jh16** [comment]
Same behavior of the override values reverting to the formula. Which currently the formula has all items go to 0.

**Comment by Alumian** [comment]
Ok thanks I'll check this out.

**Comment by Alumian** [comment]
Okay I believe that I've found and resolved the issue. From what I've found, the issue was caused by the item string. For example, consider:
`item:18423::::::::5:::::::[Head of Onyxia]|r`
`item:18423::::::::1:::::::[Head of Onyxia]|r`
The top entry is what was entered on my main character and the bottom entry was the one observed per your instructions, in Atlasloot. Apparently that particular argument is supposed to reflect the level of the character that posted the link, but that doesn't make a whole heap of sense considering the top entry was entered on my level 60 character. Regardless, I've modified the checking from comparing the item string to comparing the item ID. I've confirmed that this works on my alts who were also experiencing the issue you described.

This fix will be included in the next beta.
